src/date-input/date-input.tsx

```diff
@@ -254,56 +274,128 @@ export const DateInput = ({
     // =============================================================================
     // HELPER FUNCTIONS
     // =============================================================================
-    const performOnChangeHandler = (values: ChangeValueTypes) => {
+    const performOnChangeHandler = (value: string, isValid: boolean) => {
+        const returnValue = getFormattedValue(value, isValid);
+
         if (onChange) {
-            onChange(values);
+            onChange(returnValue);
         }
 
         if (onChangeRaw) {
-            const rawInputValues = getFormattedRawValue(values);
+            const returnRawValue =
+                DateInputHelper.getFormattedRawValue(returnValue);
 
-            onChangeRaw(rawInputValues);
+            onChangeRaw(returnRawValue);
```

Comment:
intention of `onChangeRaw` is even if user enters an invalid date like `30-02-2023`, the callback would return `["30","02","2023"]`. but since now it's derived from standalone-input's `onChange` you'd now get ["Invalid Date", "Invalid Date", "Invalid Date"] instead

perhaps you can expose an `onChangeRaw` from standalone-input as well

Reply:
Note: current i was return an object because we need to indicate startDate and endDate. Let me know if require to return an array type, please.

```
{
  start: {year: "", month: "", day: ""},
  end: {year: "", month: "", day: ""},
}
```